tests/tests/eth_provider.rs

```diff
@@ -264,6 +264,24 @@ async fn test_get_code(#[future] counter: (Katana, KakarotEvmContract), _setup:
     assert_eq!(bytecode, Bytes::from(expected));
 }
 
+#[rstest]
+#[awt]
+#[tokio::test(flavor = "multi_thread")]
+async fn test_get_logs(#[future] katana: Katana, _setup: ()) {
```

Comment:
since we can't use hive, could we develop a more powerful log test?

Reply:
sure let's make an issue, I think this already fixes a few problems

Reply:
Yes would agreed, I would say that as a follow up good first issue we could mimic a test that would be close to the hive.

Reply:
raised! https://github.com/kkrt-labs/kakarot-rpc/issues/1055